pkg/content/file.go

```diff
@@ -1,45 +1,33 @@
-/*
```

Comment:
Are these headers removed intentionally?

Reply:
No, not at all. This was more an artifact of so many variants on the file (including some fresh writes) until it got here. I will open a clean PR to add it back.

Reply:
Resolved in #21